tools/shared/src/main/scala/org/scalajs/core/tools/io/VirtualFiles.scala

```diff
@@ -132,7 +133,14 @@ trait VirtualSerializedScalaJSIRFile extends VirtualBinaryFile with VirtualScala
       ir.InfoSerializers.deserialize(stream)
     } catch {
       case e: IOException =>
-        throw new IOException(s"Failed to deserialize info of $path", e)
+        val pre = e match {
+          case e: IRVersionNotSupportedIOException =>
+            s"Requires Scala.js ${e.version} (or compatible version).\nCaused by: "
+
+          case _ => ""
+        }
+
+        throw new IOException(s"${pre}Failed to deserialize info of $path", e)
```

Comment:
Oh I see... I think it is a better idea to format the error message the way you like in the sbt plugin. (Most sbt operations that fail, don't fail with a stacktrace...